src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss

```diff
@@ -220,3 +223,54 @@ hr {
 .width-150 {
     width: 150px;
 }
+.circle-block {
+    color: #575757;
+    display: flex;
+    align-items: center;
+    :first-child {
```

Comment:
Forgot &?

Reply:
Just confirmed, it's right.

Reply:
.circle-block {
    color: #575757;
    display: flex;
    align-items: center;
    >div:first-child {
        display: inline-block;
        border-radius: 50%;
        height: 20px;
        width: 20px;
        line-height: 20px;
        font-size: 14px;
        text-align: center;
    }
}

Reply:
Modified